src/main/java/dk/kudishin/simpletasklist/services/TaskService.java

```diff
@@ -12,7 +12,7 @@ public interface TaskService {
 
     void createTask(Task t);
 
-    void updateTask(int id, Task t);
```

Comment:
I think in this case additional `id` parameter is not needed since we take it either way from `Task`.